### ISSUE TITLE
Improve the Who's using xCAT section of the landing page

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat2.github.io#28]
Looking at todays screen capture... 

![image](https://user-images.githubusercontent.com/10049070/56427418-c82dd800-6289-11e9-82c3-e4d1c305398a.png)

I'd like to request the following improvements: 
1. Ability to get to the "Who's using xCAT" section via a URL, something like: https://xcat2.github.io/#who-is-using-xcat
2. IBM and employees are a major maintainer of the project, i would list it before CINECA
3. Why did we go with a blue/white logo for IBM? The ibm.com page currently using black/white
4. The logo for CINECA has white background here, or maybe no color, why did we put a blue background on it?

**Comment by bybai** [comment]
Hi @whowutwut,
I answered about 3 in https://github.ibm.com/xcat2/team_process/issues/71.
I improved index page about 1,2,4 in PR #30.
For 4,  CINECA has transparent background and white word on it,  if we use white background or light color background, the word on the logo is not clear, so I set a blue background for it. I did not make any change of its logo, just using online logo in PR #30. 

I'd like to use white or blue color and the same size about background grid for each logo, since different logo has different size and color, the organized background grid let them look organized.
